recipes/quantization.md

```diff
@@ -1,13 +1,43 @@
 # Quantization and Sparsity
 
-torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) for more details.
+torchtune integrates with [torchao](https://github.com/pytorch-labs/ao/) for architecture optimization techniques including quantization and sparsity. Currently only some quantization techniques are integrated, see the docstrings in the [quantization recipe](quantize.py) and the [QAT recipe](qat_distributed.py) for more details.
 
 #### Quantize
 To quantize a model (default is int4 weight only quantization):
 ```
 tune run quantize --config quantization
 ```
 
+#### Quantization-Aware Training (QAT)
+
+(PyTorch 2.4+)
+
+QAT refers to applying fake quantization to weights and/or activations during finetuning,
+which means simulating only the quantization math without actually casting the original
+dtype to a lower precision.
+
+```
+tune run qat_distributed --config qat
+```
+
+This produces an unquantized model in the original data type. To get an actual quantized model,
+follow this with `tune run quantize` while specifying the same quantizer in the config.
+
+The `qat_distributed` recipe is also compatible with any full finetuning config,
+as long as a QAT quantizer is specified in the config:
```

Comment:
Sort of. But strictly speaking we associate a given config with only a single recipe. In that case we technically don't support something like e.g.

```
tune run --nproc_per_node 4 qat_distributed --config mistral/7B_full \
quantizer=torchtune.utils.quantization.Int8DynActInt4WeightQATQuantizer \
quantizer.groupsize=256
```

(This is kinda related to my other comment about how we want to actually expose QAT configs)